conduit/plugins/importers/algod/algod_importer.go

```diff
@@ -63,11 +63,12 @@ var algodImporterMetadata = plugins.Metadata{
 }
 
 func (algodImp *algodImporter) OnComplete(input data.BlockData) error {
-	if algodImp.mode != followerMode {
-		return nil
+	if algodImp.mode == followerMode {
```

Comment:
These are non-functional changes correct? Why were the existing unit tests passing for archival nodes before and now return a 404?

Reply:
This change is non functional. The one below is the real fix.